src/plot_api/plot_api.js

```diff
@@ -2508,6 +2527,360 @@ Plotly.relayout = function relayout(gd, astr, val) {
 };
 
 /**
+ * Transition to a set of new data and layout properties
+ *
+ * @param {string id or DOM element} gd
+ *      the id or DOM element of the graph container div
+ */
+Plotly.transition = function(gd, data, layout, traceIndices, transitionConfig) {
```

Comment:
I've noticed some odd behavior with non-zero `delay`:

![gifrecord_2016-08-24_113400](https://cloud.githubusercontent.com/assets/6675409/17937075/e3e2b642-69ee-11e6-96f4-fa77adb8c1fb.gif)

with 

```js
Tabs.getMock('animation', (_, mock) => {
  var gd = Tabs.fresh();
  var transitionOpts = {
    duration: 500, 
    delay: 500
  };

  Plotly.plot(gd, mock.data, mock.layout);
  Plotly.addFrames(gd, mock.frames);
 
  mock.frames.forEach((f) => {
    Plotly.animate(gd, f.name, transitionOpts)
  });
}).
```

Is this a known issue?